components/HomepageSection/Services.jsx

```diff
@@ -23,13 +23,15 @@ const Item = ({ service, category }) => {
       <SectionWrapper
         customClasses="rounded-t-xl border-t-0 border-b"
       >
-        <Image
-          src={`/images/${category}/${service.iconFilename}`}
-          alt={service.name}
-          width={300}
-          height={300}
-          className="mx-auto p-2"
-        />
+        <div style={{ width: "100%", height: 200, display: "flex" }}>
```

Comment:
For the sake of consistency, I'd suggest doing this with Tailwind utility classes.